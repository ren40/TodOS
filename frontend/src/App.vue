<template>
  <v-app>
    <v-main>
      <to-do></to-do>
    </v-main>
  </v-app>
</template>

<script>
import ToDo from "./views/ToDoViews.vue";

export default {
  name: "App",

  components: {
    ToDo,
  },

  data: () => ({
    //
  }),
};
</script>
