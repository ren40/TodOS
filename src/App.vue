<template>
  <v-app>
    <to-do></to-do>
  </v-app>
</template>

<script>
import ToDo from "./views/ToDoViews.vue";

export default {
  name: "App",

  components: {
    ToDo,
  },

  data: () => ({
    //
  }),
};
</script>
