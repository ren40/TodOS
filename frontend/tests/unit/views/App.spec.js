import App from "@/App";
import { createLocalVue, mount } from "@vue/test-utils";
import Vuetify from "vuetify";
const localVue = createLocalVue();

function addElemWithDataAppToBody() {
  const app = document.createElement("div");
  app.setAttribute("data-app", true);
  document.body.append(app);
}

addElemWithDataAppToBody();

describe("App.vue", () => {
  let wrapper;
  let vuetify;
  beforeEach(() => {
    vuetify = new Vuetify();
  });

  it("Testing the rendering of the App component", () => {
    wrapper = mount(App, {
      localVue,
      vuetify,
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
