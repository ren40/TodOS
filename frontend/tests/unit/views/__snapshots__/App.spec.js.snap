// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue  1`] = `
<div data-app="true" class="v-application v-application--is-ltr theme--light" id="app">
  <div class="v-application--wrap">
    <main class="v-main" style="padding: 0px 0px 0px 0px;">
      <div class="v-main__wrap">
        <div>
          <!---->
        </div>
      </div>
    </main>
  </div>
</div>
`;
